plugin/trino-oracle/src/main/java/io/trino/plugin/oracle/OracleClientModule.java

```diff
@@ -55,6 +56,7 @@ public void configure(Binder binder)
         configBinder(binder).bindConfig(OracleConfig.class);
```

Comment:
Use `SQLRecoverableException` as it was for retrying

Reply:
Applied:
- default is SQLTransient...
- overrident for Oracle by SQLRecoverable...